packages/dev/core/src/FlowGraph/types.ts

```diff
@@ -0,0 +1 @@
+export type AnyType = any;

```

Comment:
why is this needed? why not use any (if the usecase requires it)?

Reply:
It is used in only one place, the LogBlock (since the message can be of any type anyway) - but I can remove it yeah